docs/src/administration/web/configure-project.md

```diff
@@ -9,10 +9,19 @@ description: |
 
 ## General
 
-From the first page of the project settings, `General`, you can update the project name, or navigate to other project settings options on the left side of the screen.
+From the first page of the project settings, `General`, you can update the project name, set the project timezone, and navigate to other project settings options on the left side of the screen.
 
 ![configure project](/images/management-console/settings-basics.png "0.7")
 
+The project timezone affects [automated backups](../../dedicated/overview/backups.md).
+Changes on the automated backup schedule might affect your own maintenance window.
+
+The project timezone doesn't affect:
+
+- [App runtime](../../configuration/app/timezone.md).
+- [Crons](../../configuration/app/app-reference.md#crons) and their subsequent tasks.
```

Comment:
```suggestion
- [Cron jobs](../../configuration/app/app-reference.md#crons).
```

I don't believe tasks are something that happen after crons. Cron jobs are the tasks themselves, no?